projects/2DoBox-Pivot.md

```diff
@@ -0,0 +1,160 @@
+2: Your application has some duplication and minor bugs. Developer can speak to most choices made in the code and knows what every line is doing.
+1: Your application has a significant amount of duplication and one or more major bugs. Developer cannot speak to most choices and does not know what every line of CSS/Sass is doing. Developer writes code with unnecessary selectors or tags which do not increase clarity.
+
+## JavaScript Style
+
+4 - Application has exceptionally well-factored code with little or no duplication and all components separated out into logical components. There zero instances where an instructor would recommend taking a different approach.
+3- Application is thoughtfully put together with some duplication and no major bugs. Developer can speak to choices made in the code and knows what every line of code is doing.
+2 - Your application has some duplication and minor bugs. Developer can speak to most choices made in the code and knows what every line is doing.
+1 - Your application has a significant amount of duplication and one or more major bugs. Developer cannot speak to most choices and does not know what every line of code is doing.
+0 - Your client-side application does not function or the application does not make use of localStorage for updating information on the client. Developer writes code with unnecessary variables, operations, or steps which do not increase clarity.
+
+## Testing
+
+4 - Project has a running test suite that exercises the application at multiple levels (feature and unit). The test suite covers almost all aspects of the application.
+3 - Project has a running test suite that tests and multiple levels but fails to cover some features. All functionality is covered by tests. The application makes some use of feature testing.
+2 - Project has sporadic use of tests and multiple levels. The application contains numerous holes in testing and/or many features are untested.
+1 - There is little or no evidence of testing in this application.
+
+## Workflow
+
+4 - The developer effectively uses Git branches and many small, atomic commits that document the evolution of their application.
+3 - The developer makes a series of small, atomic commits that document the evolution of their application. There are no formatting issues in the code base.
+2 - The developer makes large commits covering multiple features that make it difficult for the evaluator to determine the evolution of the application.
+1 - The developer committed the code to version control in only a few commits. The evaluator cannot determine the evolution of the application.
+0 - The application was not checked into version control.
+
+## Code Sanitation
+
+The output from a code sanitizer (either JSHint or ESLint) shows…
+
+4 - Zero complaints
+3 - Five or fewer complaints
+2 - Six to ten complaints
+1 - More than ten complaints
+
+## Design
```

Comment:
Should we not grade this one on Design, since it's a pivot project? If they already started with a poorly designed project, it would be hard to overhaul that AND get the requirements done